package.json

```diff
@@ -24,11 +24,13 @@
     "node": ">=4.0"
   },
   "devDependencies": {
+    "avsc": "^5.0.3",
```

Comment:
We chatted about this internally, and after a bit of investigation, opted not to let this dependency come into the project without some really strong motivation.

Check out the size of `build/dist/prebid.js` when you run `gulp build --modules=adomikAnalyticsAdapter` vs `gulp build --modules=appnexusAnalyticsAdapter`. Even after minification, the first one is 147K, while the second is 61K. Since you're not adding very much code, this difference is mainly due to the `avsc` library.

On the average mobile phone, it takes ~2 seconds [just to _parse_](https://medium.com/reloading/javascript-start-up-performance-69200f43b201) 1MB of javascript, so we don't want to expose publishers to that type of performance penalty.